apisix/cli/file.lua

```diff
@@ -223,7 +223,12 @@ function _M.read_yaml_conf(apisix_home)
         return nil, "invalid config-default.yaml file"
     end
 
-    local_conf_path = profile:yaml_path("config")
+    local customized_conf_path = profile:customized_config_path()
+    if customized_conf_path then
+        local_conf_path = customized_conf_path
```

Comment:
instead of creating another variable `customized_conf_path`
Can you not directly do here `local_conf_path = profile:customized_config_path()`